### PR TITLE
Moved SerialNumberPattern files into BF telephony repo

### PR DESCRIPTION
**Pull request by tsuwandy**

### CODE REVIEW

File: experimental/SerialNumPostProcessingSkillSample/SerialNumberAction/Microsoft.Bot.Components.Telephony.csproj [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <Content Include="**/*.dialog" />
+    <Content Include="**/*.lg" />
+    <Content Include="**/*.lu" />
+    <Content Include="**/*.schema" />
+    <Content Include="**/*.uischema" />
+    <Content Include="**/*.qna" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Schemas\Microsoft.Telephony.BatchFixedLengthInput.schema" />
```
Comment (goldenwitch) [inline experimental/SerialNumPostProcessingSkillSample/SerialNumberAction/Microsoft.Bot.Components.Telephony.csproj:40]: Are these actually in this project still?

File: experimental/SerialNumPostProcessingSkillSample/SerialNumberTelephony/Readme.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+# Microsoft.Bot.Components.Telephony
```
Comment (goldenwitch) [inline experimental/SerialNumPostProcessingSkillSample/SerialNumberTelephony/Readme.md:1]: Readme lingered from other package

**Review by goldenwitch — CHANGES_REQUESTED** [review]

File: experimental/SerialNumPostProcessingSkillSample/SerialNumberAction/Microsoft.Bot.Components.Telephony.csproj (hunk 1 above)
Comment (tsuwandy) [reply experimental/SerialNumPostProcessingSkillSample/SerialNumberAction/Microsoft.Bot.Components.Telephony.csproj:40]: Nice catch!  Have not have to deal with adding/removing manually with the newer .NET.

Comment (tsuwandy) [reply experimental/SerialNumPostProcessingSkillSample/SerialNumberAction/Microsoft.Bot.Components.Telephony.csproj:40]: Extra folder, removed!

**Review by goldenwitch — APPROVED** [review]

**Pull request merged**